code/datums/gamemodes/crash.dm

```diff
@@ -199,3 +199,19 @@
 		return //Things are balanced, no burrowed needed
 	xeno_job.add_job_positions(1)
 	xeno_hive.update_tier_limits()
+
+/datum/game_mode/infestation/crash/get_total_joblarvaworth(list/z_levels, count_flags)
+	. = 0
+
+	for(var/i in GLOB.human_mob_list)
+		var/mob/living/carbon/human/H = i
+		if(!H.job)
```

Comment:
Are you sure that spawned dead mob have no jobs ?

Reply:
Looking around the map and check with VV it seems like it, if its not the case then this would also be broken in the original code since I simply copied it to overwrite it